tests/integration/support/steps.go

```diff
@@ -170,13 +172,37 @@ func (c *ScenarioCtx) InTheResponseShouldMatch(argumentType string, value *gherk
 	default:
 		break
 	}
+
+	return fmt.Errorf("response for %s not equal", argumentType)
+}
+
+// ResponseShouldHaveThisExactlyNTimes represents a step in the feature file
```

Comment:
I didn't catch this before but we should be more descriptive about these method docstrings.
 
// ResponseShouldHaveThisExactlyNTimes checks that the response object has the given property exactly N times.

Let's do this for all the other step definitions too please.